lib/dictionaries/api.js

```diff
@@ -33,7 +33,21 @@ module.exports = function(opts){
   });
 
   router.put('/:scope/:uuid/dictionaries/:name.json', parameterize, authorize, function(req, res, next) {
+    var filters = {'meta.uuid': req.params.uuid, 'meta.scope': req.params.scope, name: req.params.name};
+    if (!req.body.name){
+      req.body.name = req.params.name;
+    }
+    var payload = req.body;
+
+    var handleResponse = function(dictionary) {
+      if(dictionary) {
+        res.json(serialize(dictionary));
+      } else {
+        res.sendStatus(200);
+      }
+    };
 
+    transactions.create(filters, payload).then(handleResponse);
```

Comment:
It's weird to call a create action with filters. I think we should name it somehow in which the fact that an update or create is performed. Also payload is not a very nice name for the actual content of the dictionary. I would rather call it dictionaryData or simply dictionary.